util/src/main/java/io/kubernetes/client/util/taints/Taints.java

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.util.taints;
+
+import io.kubernetes.client.openapi.models.V1Node;
+import io.kubernetes.client.openapi.models.V1NodeSpec;
+import io.kubernetes.client.openapi.models.V1Taint;
+import java.util.Iterator;
+
+public class Taints {
+  public static enum Effect {
+    NO_SCHEDULE {
+      public String toString() {
+        return "NoSchedule";
+      }
+    },
+    PREFER_NO_SCHEDULE {
+      public String toString() {
+        return "PreferNoSchedule";
+      }
+    },
+    NO_EXECUTE {
+      public String toString() {
+        return "NoExcute";
+      }
+    }
+  };
+
+  public static V1Taint findTaint(V1Node node, String key, Effect effect) {
+    for (V1Taint taint : node.getSpec().getTaints()) {
+      if (taint.getKey().equals(key) && taint.getEffect().equals(effect.toString())) {
+        return taint;
+      }
+    }
+    return null;
+  }
+
+  public static TaintsBuilder taints(V1Node node) {
+    return new TaintsBuilder(node);
+  }
+
+  public static class TaintsBuilder {
+    private V1Node node;
+
+    public TaintsBuilder(V1Node node) {
```

Comment:
```suggestion
    TaintsBuilder(V1Node node) {
```

make the constructor non-public

Reply:
done.